fvtest/compilertriltest/ArithmeticTest.cpp

```diff
@@ -145,7 +157,10 @@ INSTANTIATE_TEST_CASE_P(ArithmeticTest, Int64Arithmetic, ::testing::Combine(
     ::testing::ValuesIn(TRTest::const_value_pairs<int64_t, int64_t>()),
     ::testing::Values(
         std::make_tuple("ladd", ladd),
-        std::make_tuple("lsub", lsub) )));
+        std::make_tuple("lsub", lsub),
+        std::make_tuple("lmul", lmul),
+        std::make_tuple("ldiv", ldiv),
+        std::make_tuple("lrem", lrem) )));
```

Comment:
`ldiv` and `lrem` can't be treated the same way as other arithmetic operators because of division-by-zero problems. Combinations that result in division-by-zero need to be filtered.